ojd_daps_skills/config/extract_skills_template.yaml

```diff
@@ -0,0 +1,47 @@
+#This is a template config file - we have added definitions to parameters that you will need to modify for your own taxonomy
+
+#the relative path to the trained NER model
+ner_model_path: "outputs/models/ner_model/20220825/"
+#the relative path to where
+taxonomy_path: "path/to/formatted_taxonomy.csv"
+#the name of your own taxonomy
+taxonomy_name: "your_taxonomy_name"
```

Comment:
keeping this for now but we can get rid of it if we end up just using the config name a la #106